client/src/App.css

```diff
@@ -1,5 +1,11 @@
 .App {
```

Comment:
Keep in mind that this css is applied to the `App` component - it is application wide.